consensus/src/pipeline/execution_schedule_phase.rs

```diff
@@ -79,7 +79,7 @@ impl StatelessPipeline for ExecutionSchedulePhase {
             for b in &ordered_blocks {
                 if let Some(tx) = b.pipeline_tx().lock().as_mut() {
                     tx.rand_tx.take().map(|tx| tx.send(b.randomness().cloned()));
-                    let _ = tx.order_proof_tx.send(());
+                    tx.order_proof_tx.take().map(|tx| tx.send(()));
```

Comment:
shall we expect (assert) `take()` to return `Some`? 

same all other places.

Reply:
probably later, I had a few places that may retry in observer prototype